config/manager/controller_manager_config.yaml

```diff
@@ -4,3 +4,5 @@ webhookPort: 9443
 leaderElection:
   enabled: true
   resourceID: kmm.sigs.x-k8s.io
+worker:
+  seLinuxType: spc_t
```

Comment:
just adding seLinuxType won't help in case of seLinux issue ( as happens on my minikube). You can simply add runAsUser: 0